api/controllers/node.js

```diff
@@ -96,9 +96,10 @@ NodeController.getStatus = function(context, next) {
 
 			data = _.cloneDeep(data);
 
-			// Check if properties are null, then set it to 0
+			// Check if properties are null, then set it to lastBlock height
 			// as per schema defined for these properties in swagger
-			data.networkHeight = data.networkHeight || 0;
+			data.networkHeight =
```

Comment:
Wtf, really? We cannot use our own `height` as `networkHeight` as it makes it no longer reliable. It's better to just remove `networkHeight` field entirely.

But, maybe instead of monkey-patching we should find a real cause of that issue? 

Lets get through the flow, looks like we always get `network` that is `undefined` here:
https://github.com/LiskHQ/lisk/blob/cdd697ea761a3df3c5e52109abc551e1a388c11a/modules/node.js#L166-L167
It's because of condition `!__private.network.peers.length`, which is `true` - it calls a callback with error, so we ends with `undefined`:
https://github.com/LiskHQ/lisk/blob/cdd697ea761a3df3c5e52109abc551e1a388c11a/modules/loader.js#L843-L845

But why there are no peers? It's because here `__private.network` is set to `{ height: 0, peers: [] }`:
https://github.com/LiskHQ/lisk/blob/cdd697ea761a3df3c5e52109abc551e1a388c11a/modules/loader.js#L841

And that behavior is always like that (every request), why? It's because all peers get filtered out here:
https://github.com/LiskHQ/lisk/blob/cdd697ea761a3df3c5e52109abc551e1a388c11a/modules/loader.js#L780-L784
And those get filtered out because all peers have always `height` lower than our `height` by `1`, look at those logs:
```
[dbg] 2018-03-27 18:18:20 | Updated peer 127.0.0.1:5000 - {"height":1344}
[inf] 2018-03-27 18:18:20 | Received new block id: 11012680464919769104 height: 1345 round: 14 slot: 5806550 reward: 0
[dbg] 2018-03-27 18:18:20 | System headers - {"os":"darwin16.7.0","version":"0.0.1","wsPort":5001,"httpPort":4001,"height":1344,"nethash":"198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d","broadhash":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","minVersion":">=0.0.0","nonce":"ymmIS1KRAeygHbD1"}
```
```
[dbg] 2018-03-27 18:18:30 | Updated peer 127.0.0.1:5000 - {"height":1345}
[inf] 2018-03-27 18:18:30 | Received new block id: 15340403393154324712 height: 1346 round: 14 slot: 5806551 reward: 0
[dbg] 2018-03-27 18:18:30 | System headers - {"os":"darwin16.7.0","version":"0.0.1","wsPort":5001,"httpPort":4001,"height":1345,"nethash":"198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d","broadhash":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","minVersion":">=0.0.0","nonce":"ymmIS1KRAeygHbD1"}
```
So looks like peers data is not updated properly when we communicate with remote peers - for example we received a block from remote peer, but we updated his height (which is lower by `1` than height of block he sent us) just before receiving that block?